### ISSUE TITLE
Interpreter_Injection#ORM_Injection link is broken

### ISSUE THREAD

**Issue by peter279k** [OWASP/www-community#175]
When looking at the [SQL injection related attacks section](https://github.com/OWASP/www-community/blob/master/pages/attacks/SQL_Injection.md#related-attacks), it seems that the [Interpreter_Injection#ORM_Injection ](https://github.com/OWASP/www-community/blob/master/pages/attacks/Interpreter_Injection#ORM_Injection) link is broken.

It should be checked or removed.

**Comment by ADubhlaoich** [comment]
On it; will have a PR tomorrow or Monday.

**Comment by ADubhlaoich** [comment]
Hey! Sorry, I dropped the ball on this for a while - partially because I was unsure what to do.

If we look at [the original page](https://wiki.owasp.org/index.php/SQL_Injection), it's clear that it's been a broken link for a while, even pre-migration.

There's [another page](https://wiki.owasp.org/index.php/Testing_for_ORM_Injection_(OTG-INPVAL-007)) that covers the same content, but was part of the OWASP Web Security Testing Guide, and thus has presumably been migrated to [its new home](https://github.com/OWASP/wstg).

In short, the link should probably be removed, though in the future I'm unsure what approach should be taken with how OWASP's content has been separated.

As an aside, I can't seem to get pages to display properly via offline Jekyll server anymore. Did something change in how the bundle's meant to be run? A lot of templating-related tag warnings all come up too, but I don't know what repo to file that in.
